.circleci/config.yml

```diff
@@ -8,7 +8,7 @@ jobs:
   build_and_tests:
     docker:
       # specify the version
-      - image: cimg/go:1.21.6
+      - image: cimg/go:1.22.2
```

Comment:
Lets then use latest 22 version, just to avoid any other potential bugs.

Reply:
Done.